.github/workflows/.reusable-sast.yml

```diff
@@ -2,156 +2,227 @@ name: sast
 
 on:
   workflow_call:
+    inputs:
+      skip:
+        description: "Want to skip running certain jobs 'none', 'non-required', 'all'?"
+        type: string
+        default: "none"
+      output:
+        description: 'Output either "sarif" (GITHUB_TOKEN with security-events:write) or print results as "table" and fail on error'
+        type: string
+        required: false
+        default: 'sarif'
 
 permissions: {}
 
 jobs:
-  codeql:
+  checkov:
     runs-on: ubuntu-latest
+    if: |
+      (github.actor != 'dependabot[bot]') &&
+      inputs.skip != 'non-required' &&
+      inputs.skip != 'all'
     permissions:
       security-events: write
-      pull-requests: read
     steps:
-    - name: Checkout repository
-      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
-    - name: Initialize CodeQL
-      uses: github/codeql-action/init@47b3d888fe66b639e431abf22ebca059152f1eea # v3.24.5
-      with:
-        languages: 'python'
-    - name: Analyze
-      uses: github/codeql-action/analyze@47b3d888fe66b639e431abf22ebca059152f1eea # v3.24.5
+      - name: Checkout code
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - name: Render Helm charts
+        run: |
+          rm -rf test # remove 'test' folder from scan #TODO: fix once final
+          rm -rf tests # remove 'tests' folder from scan
```

Comment:
This is still from the private repo, I'd guess

Reply:
i dont see the point of these two lines. i removed them/